src/starknet_interface_generator/order_imports.py

```diff
@@ -22,38 +22,33 @@ def visit_CodeBlock(self, elm: CodeBlock):
         return self.extract_imports(elm)
     
     def extract_imports(self, elm):
-        empty_element = self.get_empty_element(elm)
+        empty_element = self.get_empty_element()
         all_imports: Dict[str, List] = OrderedDict()
-        temp_import_order_names = self.import_order_names + ["rest"]
 
-        all_imports = {x: [empty_element] for x in temp_import_order_names}
+        all_imports = {x: [empty_element] for x in self.import_order_names}
         code_elements = elm.code_elements
             
         for i, x in enumerate(code_elements):
             if isinstance(x.code_elm, CodeElementImport):
-                for import_order_name in temp_import_order_names:
-                    if (import_order_name not in self.import_order_names):
-                        all_imports["rest"].append(x)
-                        break
-                    elif (import_order_name in x.code_elm.path.name):
+                import_first_word = x.code_elm.path.name.split(".")[0]
+                # group additional elements if not specified in initial list
+                if import_first_word not in self.import_order_names:
+                    self.import_order_names.append(import_first_word)
+                    all_imports[import_first_word] = [empty_element]
+                for import_order_name in self.import_order_names:
+                    if (import_order_name == x.code_elm.path.name.split(".")[0]):
                         all_imports[import_order_name].append(x)
                         break
         code_elements = list(filter(lambda x: not(isinstance(x.code_elm, CodeElementImport)), code_elements))
-        all_imports = {x: all_imports[x] for x in temp_import_order_names if len(all_imports[x]) > 1}
+        all_imports = {x: all_imports[x] for x in self.import_order_names}
         
         ordered_imports = []
         for _, v in all_imports.items():
             ordered_imports += v
```

Comment:
I suggest that you also add an alphabetical order feature for imports coming from the same file package : 
```
v.sort(key=lambda x: x.code_elm.path.name if isinstance(
                x.code_elm, CodeElementImport) else "")
```

Reply:
good sugestion